SADXLVL2/MainForm.cs

```diff
@@ -2484,7 +2485,22 @@ private void missionObjectToolStripMenuItem_Click(object sender, EventArgs e)
 				}
 		}
 
-		private void exportOBJToolStripMenuItem_Click(object sender, EventArgs e)
+        private void allToolStripMenuItem1_Click(object sender, EventArgs e)
+        {
+            exportOBJ(0);
+        }
+
+        private void visibleToolStripMenuItem1_Click(object sender, EventArgs e)
+        {
+            exportOBJ(1);
+        }
+
+        private void invisibleToolStripMenuItem1_Click(object sender, EventArgs e)
+        {
+            exportOBJ(2);
+        }
+
+        private void exportOBJ(int mode)

```

Comment:
`mode` should be an enum with clearly defined modes. An integer is too ambiguous.

Reply:
Thanks for your comment. I don't really see where I should declare the enum. Could you explain a little bit more?

Reply:
Instead of using magic numbers 0, 1, and 2, these modes should be explicitly defined using an enum. At a glance, I have no idea what those numbers do.

For example, you could declare an enum `ExportMode` with the fields `All`, `Visible`, and `Invisible`, and use these in place of 0, 1, and 2 respectively.